HackYourFuture-CPH/kp-toolbox-fp-class21
# Feature/fp21 102 backend implement add tool api endpoint

# Description

This PR adds the backend for tools on the admin side. So we have the firebase authentication for admin and "add", "delete", and "update" facilities for admin. You can run the backend and send requests which include ID Token for the admin to get access to the APIs and see the results.

Fixes # (issue)

# How to test?

You can send requests which include ID Token for the admin in the header of the request to get access to the APIs and see the results.
You also need to have serviceAccountKey file in your directory. So for testing, ask me to send it to you. It will be added to Heroku configurations too.

# Checklist
